mongoose.c

```diff
@@ -4557,14 +4557,22 @@ static void rx_dhcp_server(struct mg_tcpip_if *ifp, struct pkt *pkt) {
         53, 1, msg,                 // Message type
         1,  4, 0,   0,   0,   0,    // Subnet mask
         54, 4, 0,   0,   0,   0,    // Server ID
-        12, 3, 'm', 'i', 'p',       // Host name: "mip"
         51, 4, 255, 255, 255, 255,  // Lease time
```

Comment:
This is the name we assign the client when we are the server. Where would that be useful ? What for ?
I can only imagine when we are an AP, in Wi-Fi, and even though I've written that code, it has not been uploaded, so, no use now. Even if we were, we currently serve only one client, so which name we assign to it is quite irrelevant...

I can see it being useful when we are a client, so please, do it there.

Reply:
Ahh.. Thank you. Refactored the client instead.